Northstar.Client/mod/scripts/vscripts/ui/panel_mainmenu.nut

```diff
@@ -429,29 +431,13 @@ void function UpdatePlayButton( var button )
 			}
 			else if ( button == file.mpButton )
 			{
-				// restrict non-vanilla players from accessing official servers
-				bool hasNonVanillaMods = false
-				foreach ( string modName in NSGetModNames() )
-				{
-					if ( NSIsModEnabled( modName ) && NSIsModRequiredOnClient( modName ) )
-					{
-						hasNonVanillaMods = true
-						break
-					}
-				}
-
-				if ( hasNonVanillaMods )
-					file.mpButtonActivateFunc = null
-				else
-					file.mpButtonActivateFunc = LaunchMP
+				file.mpButtonActivateFunc = LaunchMP
```

Comment:
Add an alternative to this. We don't want end-users launching multiplayer and having issues, then coming to us about them.

Reply:
Not entirely sure how I should go about it since RequiredOnClient isn't terribly helpful since you can wrap the remotes to avoid out of sync issues. If there was some addition to the mod schema that might help but of course then old mods will have problems anyway